src/Illuminate/Support/Collection.php

```diff
@@ -232,12 +232,18 @@ public function flip()
     /**
      * Remove an item from the collection by key.
      *
-     * @param  mixed  $key
+     * @param  string|array  $key
      * @return $this
      */
     public function forget($key)
     {
-        $this->offsetUnset($key);
+        if (is_array($key)) {
```

Comment:
Why not just casting the argument to an array? I would keep consistance with other functions acception either a string, either an array


        $keys = is_array($key) ? $key : [$key];

        foreach ($keys as $key) {
            $this->offsetUnset($key);
        }

        return $this;